doc/ChangeLog

```diff
@@ -1,4 +1,159 @@
 ===============================================================
+Tag name: ctsm5.1.dev13?
+Originator(s): @cenlinhe (Cenlin He,UCAR/RAL), slevis (Samuel Levis,UCAR/TSS,303-665-1310)
+Date: Fri Aug 11 14:39:06 MDT 2023
+One-line Summary: SNICAR snow albedo scheme updates
+
+Purpose and description of changes
+----------------------------------
+
+ Notes copied here from the PR #1861:
+
+ A few substantial changes in the SNICAR module for the following updated snow
+ albedo calculation features:
+ - Updated ice optical properties from Flanner et al. (2021), with multiple types
+ for ice refractive indices. The updated SNICAR database files are:
+ /glade/work/cenlinhe/NOAA_CPT_snowAER/CLM_SNICAR_data/snicar_optics_480bnd_c012422.nc
```

Comment:
@slevis-lmwg  This line and the line below could be removed if the datasets are already in the default CLM database folder.